Makefile

```diff
@@ -113,6 +113,35 @@ install: compile
 	done
 	$(MAKE) install_man
 
+check_reproducible: compile
+	$(Q) echo "==> Checking for reproducible builds..."
+	export SOURCE_DATE_EPOCH="$(shell bin/elixir -e \
```

Comment:
I think you don't need export if you pass it explicitly later :D

Reply:
the variable is not getting stored, and on the next line I end up with 
`SOURCE_DATE_EPOCH=""`

Reply:
there you go:
https://travis-ci.org/elixir-lang/elixir/jobs/484682208#L664-L669

Reply:
Oh, it probably executes each line as a separate script? So it is not like a bash script? That makes sense.

Reply:
then we can either run the whole chunk  in one command, or store in a file in /tmp the SOURCE_DATE_EPOCH

Reply:
I will try to finish this later, as I've got to go now. Thank you

Reply:
Nah, it is perfectly fine as is. My comment is more me trying to wrap my head around makefiles (it has been a while) than anything else, sorry!

Reply:
Make is a beast hard to tame. If anybody can recommend a good reading I would greatly appreciate  it